### ISSUE TITLE
airbyte-oss/airbyte chart 0.45.0 suddenly missing

### ISSUE THREAD

**Issue by michael-baraboo-cnr** [airbytehq/airbyte#16535]
## Environment
- **Airbyte version**:  0.40.2
- **OS Version / Instance**: AWS EC2
- **Deployment**: Kubernetes
- **Step where error happened**: Deploy

## Current Behavior
When we attempt to install with chart version 0.45.0, we get an error indicating it cannot be found.
After checking the index.yaml for the chart, it appears it's no longer present.
```
Error: failed to fetch https://airbytehq.github.io/helm-charts/airbyte-0.45.0.tgz : 404 Not Found
helm.go:84: [debug] failed to fetch https://airbytehq.github.io/helm-charts/airbyte-0.45.0.tgz : 404 Not Found
```

## Expected Behavior
Successful installation of chart 0.45.0. We were previously able to install the chart with the following command in our deployment script:
```
helm upgrade --install --create-namespace --namespace airbyte airbyte airbyte-oss/airbyte \
    --version 0.45.0 \
    --wait --timeout "600s" \
    --values "values-global.yaml" \
    --values "values-${environment}.yaml"
```
The above was run successfully from my machine around 2022-09-08 20:00 UTC.

## Logs
Helm logs wth `--debug` flag.
```
Error: failed to fetch https://airbytehq.github.io/helm-charts/airbyte-0.45.0.tgz : 404 Not Found
helm.go:84: [debug] failed to fetch https://airbytehq.github.io/helm-charts/airbyte-0.45.0.tgz : 404 Not Found
helm.sh/helm/v3/pkg/getter.(*HTTPGetter).get
        helm.sh/helm/v3/pkg/getter/httpgetter.go:89
helm.sh/helm/v3/pkg/getter.(*HTTPGetter).Get
        helm.sh/helm/v3/pkg/getter/httpgetter.go:42
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).DownloadTo
        helm.sh/helm/v3/pkg/downloader/chart_downloader.go:100
helm.sh/helm/v3/pkg/action.(*ChartPathOptions).LocateChart
        helm.sh/helm/v3/pkg/action/install.go:749
main.newUpgradeCmd.func2
        helm.sh/helm/v3/cmd/helm/upgrade.go:135
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.3.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.3.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.3.0/command.go:902
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:255
runtime.goexit
        runtime/asm_arm64.s:1133
```

## Steps to Reproduce
1. Setup a valid values.yaml for installing the airbyte-oss/airbyte helm chart, the chart defaults should work fairly well.
2. Attempt install with `helm upgrade --install --create-namespace --namespace airbyte airbyte airbyte-oss/airbyte --version 0.45.0`

**Comment by milesarmstrong** [comment]
I think this was caused by https://github.com/airbytehq/airbyte/pull/16131

**Comment by DustyShap** [comment]
Seeing this as well, what was your fix?

**Comment by michael-baraboo-cnr** [comment]
> Seeing this as well, what was your fix?

@DustyShap I manually patched the changes I needed onto my deployment, and haven't needed any changes to airbyte since.
I'll likely clone and host my own chart with the repo reverted prior to https://github.com/airbytehq/airbyte/pull/16131 if it's still not working when I need to get around to it next.

I attempted using the 0.42.0 version that was pushed on 2022-09-09 in a test environment when I ran into this issue initially, but hit errors on helm upgrade and even had configurations ignored (minio being deployed despite being disabling).
